setup.sh

```diff
@@ -5,18 +5,19 @@
 
 set -e
 
-cd /home/pi
-
-git clone https://github.com/maxbbraun/uvc-gadget
-cd uvc-gadget
+echo "dtoverlay=dwc2,dr_mode=otg" | tee -a /boot/firmware/config.txt
 
-make
-
-sudo cp piwebcam.service /etc/systemd/system/
-sudo systemctl enable piwebcam
+cd /home/pi
+sudo apt install -y git meson libcamera-dev libjpeg-dev
+git clone https://gitlab.freedesktop.org/camera/uvc-gadget.git
 
-sudo sed -i 's/^console=\(.*\)$/\1 modules-load=dwc2,libcomposite/' /boot/cmdline.txt
+cd /home/pi/uvc-gadget
 
-printf "\ndtoverlay=dwc2\n" | sudo tee -a /boot/config.txt
+make uvc-gadget
+cd build
+meson install
+ldconfig
 
-sudo ln -s /lib/systemd/system/getty@.service /etc/systemd/system/getty.target.wants/getty@ttyGS0.service
+chmod +x /home/pi/pisight/rpi-uvc-gadget.sh
+cp /home/pi/pisight/rpi-uvc-gadget.sh /usr/local/bin/rpi-uvc-gadget.sh
+cp /home/pi/pisight/uvc-gadget.service /etc/systemd/system/uvc-gadget.service
```

Comment:
This installs and enables the service during setup.

```suggestion
+sudo cp /home/pi/pisight/rpi-uvc-gadget.sh /usr/local/bin/rpi-uvc-gadget.sh
+sudo cp /home/pi/pisight/uvc-gadget.service /etc/systemd/system/uvc-gadget.service
+sudo systemctl daemon-reload
+sudo systemctl enable uvc-gadget.service
+sudo systemctl start uvc-gadget.service
```